src/lib/checkbox/checkbox.ts

```diff
@@ -210,6 +215,10 @@ export class MatCheckbox extends _MatCheckboxMixinBase implements ControlValueAc
       .subscribe(focusOrigin => this._onInputFocusChange(focusOrigin));
   }
 
+  ngAfterViewChecked() {
```

Comment:
There is actually one lint error because `AfterViewChecked` interface isn't applied. Feel free to add merge ready when done.